cli/config/d2ConfigDefaults.typescript.js

```diff
@@ -0,0 +1,68 @@
+const defaultsApp = {
```

Comment:
I don't think this file is needed -- these are more JS code to import as defaults, rather than boilerplate to be copied into a new app

(I actually meant to add a comment about this when separating these objects from the boilerplate files, but forgot to add it in the last PR, so it's in a more recent one: [d2ConfigDefaults.js](https://github.com/dhis2/app-platform/pull/867/files#diff-be9857ee2a9ff24deebd7b2ba307a19368a54f46792f7cb1f735fcea95dfe6de))